nifi-frontend/src/main/frontend/apps/update-attribute/src/app/pages/update-attribute/ui/rule-listing/rule-listing.component.html

```diff
@@ -0,0 +1,187 @@
+                            <mat-expansion-panel-header
+                                class="reorder-header"
+                                [collapsedHeight]="'42px'"
+                                [expandedHeight]="'42px'">
+                                <mat-panel-title>
+                                    <div class="flex justify-between items-center">
+                                        <span class="grip -mt-0.5 mr-5 cursor-grab"></span>
+                                        <div class="flex-1 truncate">
+                                            {{ rule.name }}
+                                        </div>
+                                    </div>
+                                </mat-panel-title>
+                                <mat-panel-description>
+                                    {{ rule.comments }}
+                                </mat-panel-description>
+                            </mat-expansion-panel-header>
+                            <edit-rule
+                                [id]="rule.id"
+                                [name]="rule.name"
+                                [existingRuleNames]="getExistingRuleNames()"
+                                [comments]="rule.comments"
+                                [conditions]="rule.conditions"
+                                [actions]="rule.actions"
+                                [editable]="false"></edit-rule>
+                        </mat-expansion-panel>
+                    }
+                </mat-accordion>
+            } @else {
+                <mat-accordion [multi]="true">
+                    @for (rule of filteredRulesList; track rule.id) {
+                        <mat-expansion-panel #expansionPanel (opened)="panelOpened(rule)" (closed)="panelClosed(rule)">
+                            <mat-expansion-panel-header
+                                class="editable-header"
+                                [collapsedHeight]="'42px'"
+                                [expandedHeight]="'42px'">
+                                <mat-panel-title>
+                                    {{ rule.name }}
+                                </mat-panel-title>
+                                <mat-panel-description>
+                                    <div class="flex justify-between items-center">
+                                        <div class="flex-1 truncate">
+                                            {{ rule.comments }}
+                                        </div>
+                                        @if (isEditable) {
+                                            <button
+                                                mat-icon-button
+                                                type="button"
+                                                [matMenuTriggerFor]="actionMenu"
+                                                (click)="$event.stopPropagation()"
```

Comment:
If the user is using the keyboard and tabs to the mat-icon-button and presses the space or the enter key the menu should open not the expansion panel.

```suggestion
                                                (click)="$event.stopPropagation()"
                                                (keydown)="$event.stopPropagation()"
```